test/functional/settlementAidOfferPage.js

```diff
@@ -225,7 +225,7 @@ settlementAidOfferPage.prototype = Object.create({}, {
     // TODO: Refactor this here and in the HTML/CSS for multiple private loans?
     privateLoanAmount: {
       get: function() {
-        return element( by.id( 'contrib__private-loan' ) );
+        return element( by.css( '[data-private-loan] [data-private-loan_key="amount"]' ) );
```

Comment:
And yay, updates to the functional test page model! :smile: Can we get the functional test spec updated as well? Right now, none of this work is being verified through actual tests. (As offered before, I can pair with you on getting this done if it makes it easier.)